lib/semian/adapter.rb

```diff
@@ -53,7 +59,11 @@ def semian_options
       return @semian_options if defined? @semian_options
 
       options = raw_semian_options
-      @semian_options = options && options.map { |k, v| [k.to_sym, v] }.to_h
+
+      symbolized_options = options && options.transform_keys(&:to_sym) # rubocop:disable Style/SafeNavigation
```

Comment:
Rubocop was incorrectly flaggin this line with:

> Style/SafeNavigation: Use safe navigation (&.) instead of checking if an object exists before calling the method.

But that wouldn't work when `options` is `false`.